app/views/users/_map.html.erb

```diff
@@ -0,0 +1,49 @@
+<script src="https://maps.googleapis.com/maps/api/js?key=<%=ENV['MAP_KEY']%>&"></script>
+
+    <script src="https://maps.googleapis.com/maps/api/js?key=<%=ENV['MAP_KEY']%>&callback=initMap"
+        async defer></script> 
```

Comment:
The Geocoder for finding the lat and long of an address and rendering a map require separate google api keys.